site/docs/hive.md

```diff
@@ -121,3 +121,11 @@ Columns from the Hive SQL `SELECT` clause are projected down to the Iceberg read
 
 #### Hive Query Engines
 Both the Map Reduce and Tez query execution engines are supported.
+
+#### Hive Multi-table inserts
+It is possible to issue inserts targeting multiple Hive tables backed by Iceberg. The users should be aware that the commits are atomic only on table level, and the commit is not atomic on query level. The commits to Iceberg tables happen 1-by-1, and if one of the commit fails the other writes will not be rolled back. Example:
```

Comment:
I have a slight suggestion here but be free to ignore it, just a rewording.

Instead of "The users should be"

"Multi-table inserts will not be atomic and are committed one table at a time.  Partial changes will be visible during the commit process and failures can leave partial changes committed. Changes within a single table will remain atomic."

But feel free to ignore. I just wanted to move the important part closer to the beginning of the paragraph.

Reply:
Seems like a much better description than my one. Updated the PR.

Thanks,
Peter